Assets/MRTK/Extensions/HandPhysicsService/package.json

```diff
@@ -26,5 +26,25 @@
       },
     "dependencies": {
         "com.microsoft.mixedreality.toolkit.foundation": "%version%"
-    }
+    },
+    "files": [
+        "Interfaces*",
```

Comment:
As an alternative, so we don't have to update this if other folders / files are added, can we use [an .npmignore](http://npm.github.io/publishing-pkgs-docs/publishing/the-npmignore-file.html) for this folder?

Reply:
let me give that a shot

Reply:
done